### PR TITLE
Added test controller and DELETE function to db interface

### PR DESCRIPTION
**Pull request by garyfarb**
Added the test controller API.

To view changes, go to the app (first one) directory and run:
`docker compose up`
*If you haven't built yet, before running docker compose up, run:
`docker compose build`

Visit:
[localhost:8080](url) to access the database and view the changes live
and
[http://localhost:5050/apidocs/](url) to play around with the API.

When using "PUT" to update batch test data:
`{
  "filters": {
    "<Filters you want to select>": "<Value from database>"
  },
  "update": {
    "<Filter you want to update>": "<Value you want that filter to update to>"
  }
}`

*In the code I had to use `acronym` for ID, but on the database it will show up as ID.*

Thanks!

### CODE REVIEW

File: app/resources/tests.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+class Tests(Resource):
+  @marshal_with(test_fields)
+  def get(self):
+    """
+      Get all tests
+      ---
+      responses:
+        201:
+          description: A list of user items
```
Comment (cainsusk) [inline app/resources/tests.py:19]: change "user items" to "test items"

Comment (cainsusk) [reply app/resources/tests.py:19]: also, maybe this function should accept filters too, in order to search for tests with specific "measures" or "age" etc.

**Review by cainsusk — COMMENTED** [review]
Overall looks good ! I think the only thing we need to add is the ability to filter when getting many tests

**Comment by cainsusk** [comment]
also add tags to your docs -- Amanda suggested it for mine and it makes the docs a lot easier to read (I wish it did it by resource automatically, but alas ;-;)

**Review by farahwamer — APPROVED** [review]
Everything is working great! But can you explain to me what the following endpoints are for, I was a little confused by the descriptions (sorry if its obvious)
 * PUT /tests (update tests with batch data and filters)
 * POST /users/{id} (Get optional columns from users satisfying filters)
 * PUT /users/{id} (update with data at filters)

**Review by manda-02 — APPROVED** [review]
Everything looks great to me! Just see Farah's comment:)

**Pull request merged**